packages/calypso-e2e/src/lib/components/site-select-component.ts

```diff
@@ -15,14 +14,18 @@ export class SiteSelectComponent {
 	/**
 	 * Creates an instance of the Site Select component.
 	 *
+	 * The site selector can be shown on either one of the following:
+	 * 	- in a pseudo-sidebar, replacing the navigation sidebar.
```

Comment:
Small nit - but we should pull this line out now!

Reply:
No, it should stay - the site selector _does_ have two forms on Calypso as mentioned:

'pseudo-sidebar'
![image](https://user-images.githubusercontent.com/6549265/130655181-15b38807-d1da-4d35-9ff3-82877064ed39.png)

standalone component 
![image](https://user-images.githubusercontent.com/6549265/130655233-60f9bca5-dfa5-4974-b925-abda459a6f36.png)

Reply:
I've checked that this component works for both forms of the `SiteSelectoComponent`, so the comment does apply here.